src/js/parsers/post.js

```diff
@@ -0,0 +1,19 @@
+import Post from 'content-kit-editor/models/post';
+import SectionParser from 'content-kit-editor/parsers/section';
+import { forEach } from 'content-kit-editor/utils/array-utils';
+
+export default {
+  parse(element) {
+    const post = new Post();
+
+    forEach(element.childNodes, child => {
+      post.appendSection(SectionParser.parse(child));
```

Comment:
Hm, right so this requires a pre-processing step to the DOM before it can accept input from paste or upon page-load.